content/webapp/services/prismic/transformers/articles.ts

```diff
@@ -7,10 +7,38 @@ import {
   parseLabelType,
   parseSingleLevelGroup,
 } from '@weco/common/services/prismic/parsers';
-import { parseContentLink } from '@weco/common/services/prismic/articles';
 import { parseSeason } from '@weco/common/services/prismic/seasons';
 import { parseArticleSeries } from '@weco/common/services/prismic/article-series';
 import { london } from '@weco/common/utils/format-date';
+import {
+  isFilledLinkToDocumentWithData,
+  isFilledLinkToWebField,
+} from '../types';
+import { LinkField } from '@prismicio/types';
+import { MultiContent, transformMultiContent } from './multi-content';
+import { Weblink } from '@weco/common/model/weblinks';
+
+export function transformContentLink(
+  document?: LinkField
+): MultiContent | Weblink | undefined {
+  if (!document) {
+    return;
+  }
+
+  if (isFilledLinkToWebField(document)) {
+    return document.url
+      ? {
+          type: 'weblinks',
+          id: document.url,
+          url: document.url,
+        }
+      : undefined;
+  }
+
+  if (isFilledLinkToDocumentWithData(document)) {
```

Comment:
This is slightly different – previously we were calling `transformMultiContent()` directly on the document, but TS won't let us do that unless it knows this link is actually populated. It amounts to the same thing, and IMO is actually clearer.